### PR TITLE
Ensure ClamAV network policy tracks apps when Terraform replaces them

### PR DESCRIPTION
**Pull request by mogul**

### CODE REVIEW

**Review by rahearn — APPROVED** [review]
Approved, though if there is a link to an issue for us to check in the future for when we might be able to (or need to) remove this please add it into this comment before merging.

File: clamav/main.tf [hunk 1]
```diff
@@ -36,7 +36,14 @@ resource "cloudfoundry_app" "clamav_api" {
...
   policy {
     source_app      = data.cloudfoundry_app.app.id
-    destination_app = cloudfoundry_app.clamav_api.id
+    # We use the "id_bg" attribute here to ensure the network policy is updated
+    # during blue-green deploys of the app. Docs:
+    # https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest/docs/resources/app#update-resource-using-blue-green-app-id
+    # Note that this will probably not be necessary once the app resource
+    # supports CAPI v3; v3 has a native "--strategy rolling" flag for deploys
+    # which doesn't change the app GUID. That makes the "venerable" blue-green
+    # method obsolete.
```
[the remark this answers is not in the document]
Comment (mogul) [reply clamav/main.tf:45]: Not specifically... They have been migrating resources one at a time when people have the energy.

Although, hmmmm, [there's reference to `rolling` and v3 in this commit](https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/pull/453)... Maybe they already did it and just didn't update the docs...?

If that's the case, they might have kept the `id_bg` attribute around for backward compatibility and to keep that `venerable` strategy working.

[the remark this answers is not in the document]
Comment (mogul) [reply clamav/main.tf:45]: [WHY HELLO THERE](https://github.com/Thanhphan1147/terraform-provider-cloudfoundry/blob/93a092cd42258965963de5c1ee4e9aa243b34397/cloudfoundry/managers/v3appdeployers/rolling_strategy.go)

[the remark this answers is not in the document]
Comment (mogul) [reply clamav/main.tf:45]: https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/pull/482

**Pull request merged**